tasks/main.yml

```diff
@@ -35,10 +35,9 @@
   tags: rhosts
 
 - include: yum.yml
-  when: ansible_os_family == 'RedHat' or ansible_os_family == 'Oracle Linux'
+  when: ansible_os_family == 'RedHat'
```

Comment:
Yeah - some old pre-1.9 versions might need both but given you're about to drop even 1.9 support, hopefully this is OK.